src/web/pages.h

```diff
@@ -198,11 +198,10 @@ class configLoad : public WebRequestHandler {
     std::vector<ConfigValue> dbEntries;
     std::map<std::string, pugi::xml_node*> allItems;
     void createItem(pugi::xml_node& item, const std::string& name, config_option_t id, config_option_t aid);
-    static void setValue(pugi::xml_node& item, bool value);
+    template <typename T>
+    static void setValue(pugi::xml_node& item, const T value);
     static void setValue(pugi::xml_node& item, const std::string& value);
-    static void setValue(pugi::xml_node& item, int value);
-    static void setValue(pugi::xml_node& item, unsigned int value);
-    static void setValue(pugi::xml_node& item, size_t value);
+    static void setValue(pugi::xml_node& item, const fs::path value);
```

Comment:
You don't have to declare this here. It works as a template override with `template<>` before the definition as a posted

Reply:
Removed.